concepticondata/references/references.bib

```diff
@@ -2080,11 +2080,11 @@ @Book{Tomasello1992
   Year                                = {1992}
 }
 
-@MISC{marrison1967,
-  Title={The classification of the Naga languages of north-east India.},
+@MISC{Marrison1967,
```

Comment:
If you indicate the 'school', make it type "thesis" (or didn't this pass?) If did not pass, leave as is.

Reply:
I think this is corrected now (the type `thesis` for using the `school` field in the bibliography), but it was my suggestion to @MacyL . Strict bibtex (i.e., no biblatex) does not have a `thesis` type (and many others, particularly missing `online`) and `bibtool` complains -- the semi-official recommendation is to group everything under `misc`... 

While it would imply some work (maybe even a fork or a replacement of bibtool), I would be in favor of moving to biblatex as part of our large reworking of bibliography. Something more to discuss in the future.